packages/ir-sdk/fern/apis/ir-types-latest/definition/api.yml

```diff
@@ -1,3 +1,3 @@
 name: ir
 docs: |
- Adds support for auto-paginated endpoints.
+ Adds support for header-specific environment variables.
```

Comment:
@armandobelardo thoughts on eventually just creating a `CHANGELOG.md` for IR releases? trying to find the `api.yml` across versions is painful